web/src/main/java/org/cbioportal/genome_nexus/web/config/ApiObjectMapper.java

```diff
@@ -60,6 +61,8 @@ public ApiObjectMapper()
         mixinMap.put(IntergenicConsequences.class, IntergenicConsequencesMixin.class);
         mixinMap.put(SignalMutation.class, SignalMutationMixin.class);
         mixinMap.put(CountByTumorType.class, CountByTumorType.class);
+        mixinMap.put(IndicatorQueryResp.class, IndicatorQueryResp.class);
```

Comment:
these two lines don't seem right, aren't we supposed to map the actual class to a mixin class?

```suggestion
        mixinMap.put(CountByTumorType.class, CountByTumorTypeMixin.class);
        mixinMap.put(IndicatorQueryResp.class, IndicatorQueryRespMixin.class);
```